### ISSUE TITLE
Add support for disks/volumes customization

### ISSUE THREAD

**Issue by atsikham** [epiphany-platform/m-azure-basic-infrastructure#42]
**Is your feature request related to a problem? Please describe.**
This task is created after grooming of [another](https://github.com/epiphany-platform/epiphany/issues/139) one in epiphany classic. It's necessary to customize disks creation at least per application.

**Describe the solution you'd like**
Add an ability to customize disks/volumes.

**Describe alternatives you've considered**
None.

**Additional context**
Implementation of this task should be started after #13

**Comment by mkyc** [comment]
@atsikham shouldn't it be closed as duplicate of #28?

**Comment by atsikham** [comment]
@mkyc this task was created to have customized size and count of disks (including non-root) per VM group [#13]. #28 is only about root disk size for all VMs. IMO they can be combined and one of them can be closed.

**Comment by mkyc** [comment]
After multiple discussions today there will be no OS level mount points functionality in this module.

**Comment by rpudlowski93** [comment]
I was able to customise size and count of data disks (including non-root) per VM group.
- One VM group created with two different data disks (10GB and 30GB)
- Plan, apply and destroy process passed ✅
